apps/fz_http/lib/fz_http_web/live/connectivity_check_live/index_live.ex

```diff
@@ -6,13 +6,21 @@ defmodule FzHttpWeb.ConnectivityCheckLive.Index do
 
   alias FzHttp.ConnectivityChecks
 
+  @page_title "WAN Connectivity Checks"
+  @page_subtitle """
+  Firezone periodically checks for WAN connectivity to the Internet and logs the result here. \
+  This is used to determine the public IP address of this server for populating the default \
+  endpoint field in device configurations.
```

Comment:
Most likely not needed to `\` as HTML concatenate white spaces and treat newlines as spaces anyway.

Reply:
Yeah, I was a bit torn ;-)

I just defaulted to removing the newlines to keep existing behavior the same and the DOM a little easier to read.